src/main/java/games/strategy/triplea/printgenerator/CountryChart.java

```diff
@@ -55,10 +55,8 @@ void saveToFile(final PlayerID player, final PrintGenerationData printData) {
       }
       countryFileWriter.write("\r\n");
       // Print Unit Types
-      final Iterator<UnitType> unitIterator = gameData.getUnitTypeList().iterator();
       countryFileWriter.write(",");
-      while (unitIterator.hasNext()) {
-        final UnitType currentType = unitIterator.next();
+      for (final UnitType currentType : gameData.getUnitTypeList()) {
         countryFileWriter.write(currentType.getName() + ",");
       }
       countryFileWriter.write("\r\n");
```

Comment:
Just pointing out that there are still uses of `Iterator` in this class.  Given that this is the final part of the `Iterator` replacement PRs, I'm assuming that they were too nasty to deal with?

Reply:
Yup, I didn't bother to change iterators that had a remove call on them or used some kind of "let's modify this int while iterating and stop iterating once this int reaches a specific value" logic